client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -294,6 +340,24 @@ const PatternLargePreview = ( {
 					</ul>
 				</div>
 			) }
+			<Popover
```

Comment:
I found out that it causes the horizontal overflow 😂 We might need to set the `flip={ true }` (I'm unsure whether it works)

https://github.com/Automattic/wp-calypso/assets/13596067/5a1f525b-dc83-4adf-af92-81e693176cc3

Reply:
`flip` is true by default, unfortunately it doesn't work as expected 🙁
The horizontal overflow can be fixed by setting the position fixed.